apps/daimo-mobile/src/view/screen/receive/ReceiveScreen.tsx

```diff
@@ -46,9 +47,28 @@ function RequestScreenInner({
   const [status, setStatus] = useState<"creating" | "sending" | "sent">(
     "creating"
   );
+  const isFocused = useIsFocused();
   const nav = useNav();
   const trackRequest = useTrackRequest();
 
+  useEffect(() => {
+    let focusTimeout;
+    if (isFocused && autoFocus && Platform.OS === "ios") {
+      const { setParams }: { setParams: any } = nav;
+      setParams({
+        autoFocus: false,
+      });
+      // wait for the screen transition animation to finish before open keyboard
+      focusTimeout = setTimeout(() => {
+        textInputRef.current?.focus();
+      }, 500);
```

Comment:
do the screen transitions take 0.5sec?

Reply:
It takes 300ms but I rather take a bufor if anything loads or something happens on JS thread that would delay the animation.
Afraid of living on the edge 🫠